torch/csrc/distributed/rpc/tensorpipe_agent.h

```diff
@@ -91,6 +111,15 @@ class TensorPipeAgent : public RpcAgent {
       std::shared_ptr<FutureMessage>& futureResponseMessage,
       uint64_t messageId);
 
+  // Collects metrics from successful RPC calls
+  void trackNetworkData(
+      uint64_t requestSize,
+      uint64_t responseSize,
+      worker_id_t destRank);
```

Comment:
If we are going to remove rank, shall we try to use destName as much as possible?

Reply:
Changed this to `destWorkerId`